src/makeInfinityAdder.js

```diff
@@ -5,6 +5,21 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const adder = (a, prev = sum) => {
+    if (a === undefined) {
```

Comment:
Instead of checking if 'a' is strictly equal to 'undefined', you can use a simpler check like 'if (a == null)' which will cover both 'null' and 'undefined' and is a common pattern for such checks.